doc/api/path.markdown

```diff
@@ -94,9 +103,11 @@ Windows examples:
 
     path.isAbsolute('//server')  // true
     path.isAbsolute('C:/foo/..') // true
-    path.isAbsolute('bar\\baz')   // false
+    path.isAbsolute('bar\\baz')  // false
     path.isAbsolute('.')         // false
 
+*Note:* If the path is a zero length string then `false` will be returned.
```

Comment:
Your call - imo I would throw on this, but that is out of the picture so I'll leave it up to you.